src/main/java/teammates/ui/webapi/action/GetOngoingSessionsAction.java

```diff
@@ -61,8 +61,14 @@ public ActionResult execute() {
             throw new InvalidHttpParameterException("The filter range is not valid. End time should be after start time.");
         }
 
-        List<FeedbackSessionAttributes> allOngoingSessions =
-                logic.getAllOngoingSessions(Instant.ofEpochMilli(startTime), Instant.ofEpochMilli(endTime));
+        List<FeedbackSessionAttributes> allOngoingSessions;
+
+        try {
+            allOngoingSessions = logic.getAllOngoingSessions(Instant.ofEpochMilli(startTime),
```

Comment:
`IllegalArgumentException` is throw by which call?

Reply:
In FeedbackSessionsDb.getAllOngoingSessions, it runs: 
```
List<FeedbackSession> endEntities = load()
        .filter("endTime >", rangeStart)
        .filter("endTime <", Instant.ofEpochMilli(rangeEnd.toEpochMilli()).plus(Duration.ofDays(30)))
        .list();

List<FeedbackSession> startEntities = load()
        .filter("startTime <", rangeEnd)
        .filter("startTime >", Instant.ofEpochMilli(rangeStart.toEpochMilli()).minus(Duration.ofDays(30)))
        .list();
```
Since rangeStart and rangeEnd is created by `Instant.ofEpochMilli(startTime)` `Instant.ofEpochMilli(endTime)`, if startTime and endTime is Long.MIN_VALUE and endTime is Long.MAX_VALUE respectively, when they change by adding or removing days, it crashes with that error.

<details><summary>The full stack trace is:</summary>
<p>


```java
java.lang.IllegalArgumentException: java.lang.ArithmeticException: long overflow
	at java.util.Date.from(Date.java:1359)
	at teammates.storage.entity.BaseEntity$InstantTranslatorFactory$1.saveValue(BaseEntity.java:42)
	at teammates.storage.entity.BaseEntity$InstantTranslatorFactory$1.saveValue(BaseEntity.java:34)
	at com.googlecode.objectify.impl.translate.ValueTranslator.saveSafe(ValueTranslator.java:39)
	at com.googlecode.objectify.impl.translate.NullSafeTranslator.save(NullSafeTranslator.java:25)
	at com.googlecode.objectify.impl.ObjectifyImpl.makeFilterable(ObjectifyImpl.java:310)
	at com.googlecode.objectify.impl.QueryImpl.addFilter(QueryImpl.java:146)
	at com.googlecode.objectify.impl.QueryImpl.filter(QueryImpl.java:94)
	at com.googlecode.objectify.impl.QueryImpl.filter(QueryImpl.java:32)
	at teammates.storage.api.FeedbackSessionsDb.getAllOngoingSessions(FeedbackSessionsDb.java:44)
	at teammates.logic.core.FeedbackSessionsLogic.getAllOngoingSessions(FeedbackSessionsLogic.java:110)
	at teammates.logic.api.Logic.getAllOngoingSessions(Logic.java:286)
	at teammates.ui.webapi.action.GetOngoingSessionsAction.execute(GetOngoingSessionsAction.java:67)
	at teammates.test.cases.webapi.BaseActionTest.lambda$verifyHttpParameterFailure$2(BaseActionTest.java:428)
	at teammates.test.cases.BaseTestCase.assertThrows(BaseTestCase.java:196)
	at teammates.test.cases.webapi.BaseActionTest.verifyHttpParameterFailure(BaseActionTest.java:428)
	at teammates.test.cases.webapi.GetOngoingSessionsActionTest.testBoundaryValues(GetOngoingSessionsActionTest.java:116)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
Caused by: java.lang.ArithmeticException: long overflow
	at java.lang.Math.multiplyExact(Math.java:892)
	at java.time.Instant.toEpochMilli(Instant.java:1237)
	at java.util.Date.from(Date.java:1357)
	... 40 more
```

</p>
</details>